base/gmp.jl

```diff
@@ -518,6 +534,12 @@ iszero(x::BigInt) = x == Clong(0)
 <(i::Integer, x::BigInt) = cmp(x,i) > 0
 <(x::BigInt, f::CdoubleMax) = isnan(f) ? false : cmp(x,f) < 0
 <(f::CdoubleMax, x::BigInt) = isnan(f) ? false : cmp(x,f) > 0
+isneg(x::BigInt) = x.size < 0
+ispos(x::BigInt) = x.size > 0
```

Comment:
I think it would be useful indeed to define these functions more generally for `Integer`, in the same way that `iszero` is defined (it "may" increase performances...), but I prefer not in this PR. 
I'm still a bit reluctant with writing `ispositive` and `isnegative` out in full: `x > 0` is so much easier to write! also, what should we call `x >= 0` ? `ispositiveoreq(x)` (or `isposeq(x)`) ? I'm wondering now if it wouldn't be simpler to check for the value `0` in the implementation of `<(x::BigInt, y::Int)` etc.